packages/dds/tree/src/test/tree/visit.spec.ts

```diff
@@ -196,6 +199,39 @@ describe("visit", () => {
 		testTreeVisit(delta, expected);
 	});
 
+	it("move child to the right", () => {
```

Comment:
Is the meaningful difference between this test and the one below it that this test deals with root nodes?
If so, it would be good to rename the test:
```suggestion
	it("move roots to the right", () => {
```
If not, what code path does this test exercise that the one below it doesn't?

Reply:
```suggestion
	it("move node to the right", () => {
```

Reply:
the one below is a more complex case. It moves multiple nodes, has multiple fields, uses paths that are longer. This is unit testing move, without testing traversal, hierarchy etc.

Note that this test wasn't intended as a regression test for the bug it found: it just got lucky that it also found a bug for root editing.